config/base.json

```diff
@@ -37,6 +37,7 @@
 	},
 	"flags": {
 		"enable_datadog": true,
-		"enable_io_beta_cta": true
+		"enable_io_beta_cta": true,
+		"enable_hvd": false
```

Comment:
See this [config plugin](https://github.com/hashicorp/dev-portal/blob/main/config/plugin.js) for more context. This config data is injected at buildtime and can be read throughout the application.

Reply:
Also, the `base` config is extended in every environment so it being `false` here is what will prevent the pages from rendering in production.